docs: update help wanted stappenplan (#1804)
nl-design-system/documentatie: docs/handboek/component-bijdragen/help-wanted-stappenplan.mdx

```

**🚩 Checkpoint**: Kernteam = 'Declined'

## Status bijgewerkt naar Help Wanted

Doel: iedereen kan zien dat de component nu richting Community wil.

Deze stap kan enkel worden uitgevoerd door het kernteam.

- Voeg het 'Help Wanted' label toe aan de GitHub Discussion.
- Voeg het 'Help Wanted' label toe aan het backlog issue.
- Selecteer bij ‘Projects’ de volgende extra projecten:
  - Components - 2 - Community
  - Components - 3 - Candidate
  - Components - 4 - Hall of Fame
- Filter het [Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1) op de component door op `{naam-component}` te zoeken.
- Kopieer de url na het filteren.
- Voeg onderstaande tekst toe als comment aan de GitHub Discussion.

```md
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[{naam-component} op het Community bord]({url-community-bord})
```

**🚩 Checkpoint**: Status

## Vindbaar op de NL Design System website

Doel: Hierdoor kan de component worden opgepakt door de community.

Deze stap kan enkel worden uitgevoerd door het kernteam.

> ### Schematische afbeelding maken

Reviewer: Willen we als kernteam de verantwoordelijkheid nemen voor het maken van deze schematische afbeelding? En dus voorkomen dat anderen dit doen, aangezien zij geen bewerkingsrechten hebben in ons bestand?

In dat geval zou ik er ook bij zetten dat deze stap enkel door het kernteam kan worden gedaan.

Author: Ja dat willen we. En eigenlijk geldt dat voor alle stappen van dit stuk 'Vindbaar op de NL Design System website'. Daarom dacht ik het direct onder 'doel' 1 keer te noemen.

- Maak een schematische afbeelding in het ['Doc website - Afbeeldingen' Figma bestand](https://www.figma.com/file/0Y9Tbb373u6QGxGT6mqUnm/NLDS---Doc-website---Afbeeldingen?type=design&node-id=606-943&mode=design).
- Gebruik als naam voor het frame 'componenten_overzicht_{naam-component}.

> - Exporteer de afbeelding als svg bestand.

Reviewer: ```suggestion
- Exporteer de afbeelding als SVG-bestand.
```

### Branch klaar zetten
- Open de [documentatie repository](https://github.com/nl-design-system/documentatie) in Visual Studio Code.

> - Maak een nieuwe branch aan gebruik hiervoor de volgende naam:

Reviewer: ```suggestion
- Maak een nieuwe branch aan. Gebruik hiervoor de volgende naam:
```

**Branch**
```
feat/add-documentation-page-for-{naam-component}
```

### Schematische afbeelding bewerken en toevoegen

Doorloop de volgende stappen om de schematische afbeelding mee te laten kleuren met de statussen van het Estafettemodel.

**Tip!**
Installeer in Visual Studio Code de extentie 'SVG' ontwikkeld door Jock.

- Ga naar de [folder 'static/svg'](https://github.com/nl-design-system/documentatie/tree/main/static/svg).

> - Sleep het svg bestand in deze folder. De code van het svg bestand wordt zichtbaar.

Reviewer: ```suggestion
- Sleep het SVG-bestand in deze folder. De code van het SVG-bestand wordt zichtbaar.
```

> - Kies 'View' in de hoofd navigatie van Visual Studio Code en open 'Command Pallete...'.

Reviewer: ```suggestion
- Kies 'View' in de hoofdnavigatie van Visual Studio Code en open 'Command Palette...'.
```

> - Type 'svg' en kies voor 'Preview SVG'. De visuele weergave van het svg bestand wordt zichtbaar.

Reviewer: ```suggestion
- Typ 'svg' en kies voor 'Preview SVG'. De visuele weergave van het SVG-bestand wordt vervolgens zichtbaar.
```

- Verwissel alle kleurwaardes van de fills en strokes naar design tokens die zijn aangevuld met een fallback.
  - `#F9F6F3` wordt `var(--component-illustration-background-color, white)`.
  - `#E4EBED` wordt `var(--component-illustration-grid-color, #EEE)`.
  - `#004152` wordt `var(--component-illustration-color, #666)`.
  - `white` wordt `var(--component-illustration-background-color, white)`.

**Tip!**

> Met de toetscombinatie `Cmd+D` (of `Ctrl+D`) kun je stukjes code selecteren die overeen komen met je huidige selectie.

Reviewer: ```suggestion
Met de toetscombinatie `Cmd+D` (of `Ctrl+D`) kun je stukjes code selecteren die overeenkomen met je huidige selectie.
```

> - Voeg de volgende stukjes code toe zodat het svg bestand dynamisch kan worden ingeladen.

Reviewer: ```suggestion
- Voeg de volgende stukjes code toe om het SVG-bestand dynamisch in te laden.
```

  - Op de 2e regel, direct na `<svg...`, plak je `<symbol id="component-illustration">`.
  - Op de voorlaatste regel, vlak voor `</svg>`, plak je `</symbol>`.
  - Direct daaronder, ook weer vlak voor `</svg>`, plak je `<use href="#component-illustration"/>`.
- Maak een commit en gebruik hierbij het volgende bericht:

**Commit Message**
```
feat: add illustration for {naam-component}
```

### Documentatie pagina toevoegen

- Ga naar de [folder 'docs/componenten'](https://github.com/nl-design-system/documentatie/tree/main/docs/componenten).

> - Voeg een nieuwe folder toe voor de component, gebruik hiervoor kebab-case als schrijfwijze.

Reviewer: ```suggestion
- Voeg een nieuwe folder toe voor de component en gebruik hiervoor kebab-case als schrijfwijze.
```

> - Voeg een nieuw bestand toe en noem deze 'index.mdx'.

Reviewer: ```suggestion
- Maak een nieuw bestand aan en noem het 'index.mdx'.
```

> - Vul het bestand op basis van [de component template](https://raw.githubusercontent.com/nl-design-system/documentatie/main/docs/componenten/_template.mdx).

Reviewer: ```suggestion
- Vul het bestand in volgens [de component template](https://raw.githubusercontent.com/nl-design-system/documentatie/main/docs/componenten/_template.mdx).
```

- Maak een commit en gebruik hierbij het volgende bericht:

**Commit Message**
```
feat: add documentation page for {naam-component}
```

### Component progress bijwerken

> - Publiceer een nieuwe versie van het `component-progress` package door op `Run Workflow` te klikken op [GitHub Actions van de Index repository](https://github.com/nl-design-system/index/actions/workflows/update-component-progress.yml). Kies hierbij voor de `main` branch.

Reviewer: ```suggestion
- Publiceer een nieuwe versie van het `component-progress` package door op `Run Workflow` te klikken in [GitHub Actions van de Index repository](https://github.com/nl-design-system/index/actions/workflows/update-component-progress.yml). Kies hierbij voor de `main` branch.
```

> - Wacht tot [component-progress op npm](https://www.npmjs.com/package/@nl-design-system/component-progress) is bijgewerkt. Dat zie je door naar `Last publish` te kijken.

Reviewer: ```suggestion
- Wacht tot [component-progress op npm](https://www.npmjs.com/package/@nl-design-system/component-progress) is bijgewerkt. Je kunt dit controleren door te kijken naar de datum bij `Last publish`.
```

- Update `@nl-design-system/component-progress` in de documentatie repository. Dat doe je door lokaal in de branch `pnpm install @nl-design-system/component-progress@latest --save-dev -w` te draaien. Dit update de devDependency en de lockfile.
- Maak een commit en gebruik hierbij het volgende bericht:

**Commit Message**
```
build: update component progress
```

### Branch publiceren

- Publiceer je branch naar GitHub om een pull request (PR) aan te maken.
- Ga naar GitHub en gebruik de volgende opzet voor je PR:

**Title**
```
docs: add {naam-component} to documentation site
```

**Description**
```
- add illustration for {naam-component}.
- add documentation page for {naam-component}.
- update component progress.
```

> - Klik op 'Create Pull Request'

Reviewer: ```suggestion
- Klik op 'Create Pull Request'.
```

- Geef een developer uit het kernteam via Slack een seintje dat de PR klaar staat.

**🚩 Checkpoint**: nldesignsystem.nl

## 🏁 Finish

> Zo wat een werk! Je hebt alle stappen genomen en zo alle checkpoints behaald die nodig zijn voor de ['Help Wanted' status](https://github.com/orgs/nl-design-system/projects/27/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). De component gaat nu door voor de 'Community' status. Goed om dit ook mede te delen aan de rest van de community.

Reviewer: ```suggestion
Zo wat een werk! Je hebt alle stappen genomen en zo alle checkpoints behaald die nodig zijn voor de ['Help Wanted' status](https://github.com/orgs/nl-design-system/projects/27/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). De component gaat nu door voor de 'Community' status. Het is goed om dit ook met de rest van de community te delen.
```
